.github/workflows/docs.yaml

```diff
@@ -0,0 +1,94 @@
+#
+# This file is part of Astarte.
+#
+# Copyright 2025 SECO Mind Srl
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+name: docs
+
+on:
+  workflow_dispatch:
+  release:
+    types:
+      - published
+defaults:
+  run:
+    shell: bash
+permissions:
+  contents: read
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: false
+
+jobs:
+  latest-release:
+    runs-on: ubuntu-24.04
+    env:
+      GH_TOKEN: ${{ github.token }}
+      CURRENT_RELEASE: ${{ github.event.release.tag_name }}
+    outputs:
+      result: ${{ steps.check-release.outputs.RESULT }}
+    steps:
+      - name: Check release branch
+        id: check-release
+        run: |
+          latest_release=$(gh release --repo '${{ github.repository }}' list --json='tagName' -q='.[].tagName' | sort --version-sort | tail -n 1)
+          if [[ "$CURRENT_RELEASE" == "$latest_release" ]]; then
+            echo "Current branch is the latest release branch"
+            echo "RESULT=OK" >> "$GITHUB_OUTPUT"
+          else
+            echo "Current branch is not the latest release branch"
+            echo "RESULT=FAILED" >> "$GITHUB_OUTPUT"
+          fi
+  push-get-started:
+    runs-on: ubuntu-24.04
+    needs: latest-release
+    if: needs.latest-release.outputs.RESULT == 'OK'
+    env:
+      OUR_FILE: 'docs/get-started.md'
+      THEIR_FILE: './sdk-doc/source/get_started/rust.md'
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/checkout@v4
```

Comment:
Remove one
```suggestion
```

Reply:
The first one clones the rust repo and the second one the sdk-doc. It cannot be removed

Reply:
in this case add comment on first checkout